x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -402,6 +416,59 @@ export class AlertsClient<
     return this.legacyAlertsClient.factory();
   }
 
+  public async getPersistentAlerts({
+    ruleId,
+    spaceId,
+    excludedAlertInstanceIds,
+    alertsFilter,
+    start,
+    end,
+    executionUuid,
+    formatAlert,
+    isLifecycleAlert = false,
```

Comment:
Can we use the `autoRecoverAlerts` boolean that's set on the `ruleType` in place of passing this in? If `autoRecoverAlerts = true` then it is a lifecycle alert, otherwise it is not.

Reply:
done